src/components/AddressBookSelector.tsx

```diff
@@ -91,29 +90,27 @@ const AddressBookSelector = forwardRef(
 
     return (
       <Box flex={1} {...boxProps}>
-        <Portal>
```

Comment:
Was the portal something new? Or just not used?

Reply:
Yeah I had added this but it was causing some issues on other screens